pyvex/lifting/__init__.py

```diff
@@ -45,6 +45,12 @@ def lift(data, addr, arch, max_bytes=None, max_inst=None, bytes_offset=0, opt_le
 
     .. note:: If no instruction and byte limit is used, pyvex will continue lifting the block until the block
               ends properly or until it runs out of data to lift.
+
+    .. note:: If you specify an opt_level of 0, this will also disable many arch-specific optimizations
```

Comment:
Could this actually be changed to opt_level=-1 to trigger the big unoptimization stuff?

Reply:
I want to hear what you think about opt_level=-1...